src/Lecture2.hs

```diff
@@ -198,7 +305,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge as [] = as
+merge [] bs = bs
+merge (a : as) (b : bs)
+  | a <= b = a : merge as (b : bs)
+  | otherwise = b : merge (a : as) bs
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️